source/help/messaging/mentioning-teammates.rst

```diff
@@ -0,0 +1,56 @@
+.. _mentioning-teammates:
+
+Mentioning Teammates
+====================
+
+
+@Mentions
+---------
+
+Use @mentions to get the attention of specific team members.
+
+@username
+~~~~~~~~~
+
+You can mention a teammate by using the *@* symbol plus their username to send them a mention notification.
+
+Type *@* to bring up a list of team members who can be mentioned. To filter the list, type the first few letters of any username, first name, last name, or nickname. Use the UP and DOWN arrow keys to scroll through entries in the list, and then press ENTER to select the person to mention. When selected, the username replaces the full name or nickname.
```

Comment:
Cool, thanks Jeff, so, in effect, "and" could be left out completely - e.g.  "Use the UP and DOWN arrow keys to scroll through entries in the list, then press ENTER to select the person to mention."  0/5 on changing but just wanted to clarify standards.